package.json

```diff
@@ -6,6 +6,9 @@
   "keywords": [],
   "main": "src/index.js",
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.26",
```

Comment:
Do you need to import this, you don't seem to be using it.

Reply:
Maybe long-arrow-alt-right

Reply:
I think we need this for react-fontawesome. I got `Module not found: Can't resolve '@fortawesome/fontawesome-svg-core' in '/Users/paulomenezes/repositories/excalidraw/node_modules/@fortawesome/react-fontawesome'` when remove

Reply:
Sounds good :)